irc.py

```diff
@@ -257,13 +260,52 @@ def command(self, cmd: bytes) -> None:
             print('Unknown command: ', cmd)
 
 
+def ip2long(ip):
+    """
+   Convert an IP string to long
+   """
+    packedIP = socket.inet_aton(ip)
+    return struct.unpack("!L", packedIP)[0]
+
 
 def main():
+    ###############################
+    # t == token-filename
+    # i == IP-Address
+    # p == port-number
+    # u == no-userlist
+    ###############################
+    utoken=''
+    uip='127.0.0.1'
+    uport='9007'
+    uuser=''
+ 
+    # Read command line args
+    myopts, args = getopt.getopt(sys.argv[1:],"t:i:p:u:")
+ 
+    ###############################
+    # o == option
+    # a == argument passed to the o
+    ###############################
+    for o, a in myopts:
+        if o == '-t':
+            utoken=a
+        elif o == '-i':
+            uip=a
+        elif o == '-p':
+            uport=a
+        elif o == '-u':
+            uuser=a
+        else:
+            print("Usage: %s -t token-filename -i IP-address -p port-number -u no-userlist" % sys.argv[0])
+ 
+    nuip=ip2long(uip)
+    uip=socket.inet_ntoa(struct.pack('!L', nuip))
```

Comment:
no need in python, you can just pass the string `127.0.0.1` as I do and it works.